src/Middleware/Rewrite/src/RewriteOptionsExtensions.cs

```diff
@@ -128,6 +128,17 @@ public static RewriteOptions AddRedirectToWwwPermanent(this RewriteOptions optio
             return AddRedirectToWww(options, statusCode: StatusCodes.Status308PermanentRedirect);
         }
 
+        /// <summary>
+        /// Permanently redirects the request to the www subdomain if the request is non-www.
+        /// </summary>
+        /// <param name="options">The <see cref="RewriteOptions"/>.</param>
+        /// <param name="domains">List of domains on which the www. rule will be applied.</param>
```

Comment:
```suggestion
        /// <param name="domains">The specific list of domains to redirect to.</param>
```

Reply:
Thanks for approving the PR and helping me get this through! About the description, of I’m not convinced on either of those... any thoughts on these options: 
- Limit the rule to apply only on the specified domain(s).
- Limit the rule to apply only on requests from the specified domain(s).